casa/Json/test/tJsonKVMap.in

```diff
@@ -0,0 +1,20 @@
+# This defines the input for a file.
```

Comment:
As I said many parsers accept comments because many see the inability of having comments in Json as a major drawback. By default JsonOut will not write comments (to be fully Json compliant), but it is possible to do so.
I do not understand your comment about number formatting. Json supports integer and floating point notation which are the only number types written by JsonOut. It also supports complex numbers, but does so using a standard Json struct.